src/templates/util.rs.template

```diff
@@ -0,0 +1,4 @@
+use std::sync::Mutex;
+
+pub static REQWEST_CLIENT: Mutex<Option<reqwest::blocking::Client>> = Mutex::new(None);
+pub static REQWEST_BASE_URL: &str = "https://abcdef.de";
```

Comment:
The URL needs to be injected via the CLI or by Nazara itself. Statically changing it here does not work.

Reply:
I fixed it